### DISCUSSION TITLE
【标签功能】对于标签功能的一些建议

### DISCUSSION THREAD

**Discussion by nowscott** [tangly1024/NotionNext#2091] [Ideas]
**为什么提出这个新的特性改动**
对于目前来讲，标签和分类的功能有些同质化，但我个人认为标签的功能其实很强大的，甚至可以做到和分类协同筛选。

**描述一下你推荐的解决方案**
目前还停留在一个构想，但我觉得现在的标签没实现应有的功能，应该可以做到多选来精确筛选文章，这对文章多的博客有很大的作用。举个例子：比如数学部分和英语文章部分每个标签下都有很多文章，但如果我想精确找到某一篇用英文写的关于数学的文章，我就可以点选英语文章和数学两个标签来精确筛选。

当然可以说目前有搜索功能可以做到类似的效果，但我觉得没有使用标签筛选来的方便，或者说各有各的应用场景。

同时我也可以使用分类先将学术类的分在一个类别中，然后在学术这个分类中去利用英语文章和数学这两个标签来进一步筛选。

**Comment by tangly1024** [comment]
同时用两个标签筛选出一篇文章，很酷的想法。

不过要怎么实现，暂时没有头绪

**Comment by nowscott** [comment]
https://www.nowscott.top/
上面是我做的一个索引网站，其中网站的筛选我就大概用了这种形式，只不过这个网站中的标签我还有修改的想法。
实现的效果大概像下图这样
<img width="859" alt="截屏2023-08-03 20 05 04" src="https://github.com/tangly1024/NotionNext/assets/86339284/ffb28703-856f-4cb5-b20d-f68b6c118fac">
